packages/cfd/src/Constants/routes-config.js

```diff
@@ -23,6 +23,10 @@ const initRoutesConfig = () => {
             getTitle: () => localize('MT5'),
             is_authenticated: false,
         },
+        {
+            path: routes.compare_cfds,
+            component: CFDCompareAccounts,
```

Comment:
No need to forward route props?

Reply:
I don't think we need to forward the route props here as the existing routes are not used as DMT5 page is gone.
@hirad-deriv can you confirm this?

Reply:
We will remove this part from the cfd package in our improvement but you'd stick to the flow that we currently have.